progressbar_example.py

```diff
@@ -0,0 +1,121 @@
+import porepy as pp
```

Comment:
The placement of this example is wrong. Also, do we need an example, and not just a tutorial?